src/components/ProductDetail.vue

```diff
@@ -63,7 +63,7 @@ export default {
     },
     created(){
         let id =useRoute().params.id
-        axios.get('https://dummyjson.com/products/'+id)
+        axios.get('/'+id)
```

Comment:
`    axios.get('/${id}')`
you can use template literals like this to avoid string concatination